packages/wallet/src/TxBuilder/types.ts

```diff
@@ -0,0 +1,126 @@
+import { Cardano } from '@cardano-sdk/core';
+import { CustomError } from 'ts-custom-error';
+import { DeepPartial } from '@cardano-sdk/util';
+import { InputSelectionError } from '@cardano-sdk/cip2';
+
+import { OutputValidation } from '../types';
+import { SignTransactionOptions } from '../KeyManagement';
+
+// Does not transform TokenMap into Partial<TokenMap> because it would also make it's methods and map types optional
+// and that forces users to do many checks before being able to use any of it's properties and methods
+export type PartialTxOut = DeepPartial<Cardano.TxOut, Cardano.TokenMap>;
+
+export enum TxOutputFailure {
+  minimumCoin = 'Minimum coin not met',
```

Comment:
Let's follow the convention that other errors are already using
```suggestion
  MinimumCoin = 'Minimum coin not met',
```

Reply:
[231a55c](https://github.com/input-output-hk/cardano-js-sdk/pull/396/commits/231a55c8ae512493304f6585d6bd230b5c65ced3)